client/src/components/CreatePost/CreatePost.js

```diff
@@ -124,7 +125,7 @@ const Wrapper = ({ onCancel, visible, children }) => {
   );
 };
 
-const Step4 = () => {
+const Step4 = ({}) => {
```

Comment:
You don't need this `{}` here since you're not passing in any object properties.

Reply:
Yeah @BMariscal, I totally forgot to remove it from there. and thanks, I would check out the PR, and see to it that the conflict is resolved, meanwhile, I just got my docker running a few days ago, so i would finish this up today hopefully @idjevm

Reply:
Yeah @BMariscal,  totally forgot to remove it, i had it there, when i was testing,so i would remove in my next PR